pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -9,31 +9,34 @@
     "aws_region": "us-west-2",
     "aws_secret_key": "",
     "aws_instance_type": "t3.small",
+    "build_name": "",
```

Comment:
I took the liberty to alphabetize this section since its getting big and finding settings was getting harder to spot.